lib/tasks/prescat.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+namespace :prescat do
```

Comment:
is there something we can/should do to show this is dangerous and should only be undertaken by experts?

Similarly, should we indicate that this rake task is temporary and expected to go away?

Reply:
I honestly wouldn't expect it to go away, unless it's being replaced with same-druid-on-many-roots code. We're going to have to do these migrations every couple of years until that code gets sorted.

Reply:
i would hope we'd get support for multiple `CompleteMoab`s done in less than a couple years from now, but also i sorta expect to be putting a laugh emoji on this comment two years from now.  i don't think supporting multiple `CompleteMoab`s is actually a crazy amount of work, though maybe it's a lot of work for the amount of time we usually have.  

but i think it'd also make this sort of migration much easier since it'd prevent having to pause accessioning, because we could copy stuff to a new place, and then remove it from the old place after we've determined it's safe in its new home.

Reply:
> is there something we can/should do to show this is dangerous and should only be undertaken by experts?

current version now has a strong warning prompt

> Similarly, should we indicate that this rake task is temporary and expected to go away?

and this seems unsettled at present?